.env.default

```diff
@@ -1,3 +1,7 @@
+# The base URL for all locations
+BASE_PATH=/
+BASE_URL=$BASE_PATH
```

Comment:
we already have PUBLIC_PATH and PUBLIC_URL variables.
how can we reuse it ? we need to think about it

Reply:
Do you want to get all symbols before the last slash in PUBLIC_URL variable? Cause this variable is used for assets and looks like `/assets`.